packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## v3.1.1 (Sep 21, 2023)
+## v3.1.1-preview.0 (Sep 21, 2023)
```

Comment:
Please make sure all changelog files do not have lines mentioning the temporary preview versions.